dexter/processing/document_processor.py

```diff
@@ -207,10 +207,18 @@ def fetch_daily_feeds(self, day):
         if self.FEED_PASSWORD is None:
             raise ValueError("%s.FEED_PASSWORD must be set." % self.__class__.__name__)
 
+        # r = requests.get(self.FEED_URL % day.strftime('%d-%m-%Y'),
```

Comment:
I'd delete this code. Commenting out code just leaves a mess. It's always in git if you need it later.